docs/terms/action.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Action
+---
+
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '../images/universal_map/_um_setup_header.mdx'
+import ConnectHeader from '../images/universal_map/_um_connect_header.mdx';
+import CreateHeader from '../images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '../images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='inactive' connect='inactive' create='inactive' validate='active'/> 
+
+## Overview
+
+### Definition
+
+An Action is a Python class with a `run()` method that takes the result of validating a <TechnicalTag relative="../" tag="batch" text="Batch" /> against an <TechnicalTag relative="../" tag="expectation_suite" text="Expectation Suite" /> and does something with it.
+
+### Features and promises
+
+Actions are highly customizable.  Great Expectations comes with common Actions for such things as sending email or Slack notifications, updating <TechnicalTag relative="../" tag="data_docs" text="Data Docs" />, and storing <TechnicalTag relative="../" tag="validation_result" text="Validation Results" /> out of the box.  However, it is easy to create custom Actions by creating a subclass of Great Expectations' `ValidationAction` class and overwriting it's `_run()` method. This means that you can configure an Action to do literally anything you are capable of programming in Python in response to a <TechnicalTag relative="../" tag="checkpoint" text="Checkpoint" /> <TechnicalTag relative="../" tag="validation" text="Validation" /> completing.
+
+### Relationship to other objects
+
+Actions are executed by <TechnicalTag relative="../" tag="validation_operator" text="Validation Operators" />, which are a component of Checkpoints.  In general, Validation Operators function behind the scenes, so it is enough to know that Actions are configured inside the `action_list` parameter of a Checkpoint's configuration, and execute every time the Checkpoint finishes running a Validation.
```

Comment:
I would think that we may not want to even mention `ValidationOperators` since it could lead to some confusion. That said, if someone else feels differently, and that it is helpful to maintain the docs reference, I think the way you have it hear with the disclaimers and warnings is the right way to do it.

Reply:
I'd like to leave the reference in place purely for completeness and accuracy's sake -- I can imagine a scenario in which someone is working on extending Great Expectations in some way that requires knowledge of how Actions are executed.